src/fhir/2_CarePlan.all-qnrs-dcw.json

```diff
@@ -0,0 +1,249 @@
+{
+    "resourceType": "CarePlan",
+    "id": "CarePlan-patient-dcw-gimmethemall",
+    "description": "Hand-coded CarePlan for patient-dcw-gimmethemall",
+    "status": "active",
+    "intent": "order",
+    "category": [
+        {
+            "coding": [
+                {
+                    "system": "http://snomed.info/sct",
+                    "code": "719091000000102",
+                    "display": "Questionnaire"
+                }
+            ],
+            "text": "Questionnaire"
+        }
+    ],
+    "subject": {
+        "reference": "Patient/patient-dcw-gimmethemall"
+    },
+    "activity": [
+        {
+            "detail": {
+                "instantiatesCanonical": [
+                    "Questionnaire/CIRG-ECOG-SELF-REPORT"
```

Comment:
@mcjustin I think this should be CIRG-ECOG12 see [here](https://github.com/uwcirg/asbi-screening-app/blob/460bcabca655486f6e8a6e5678f3acef6a1ca4d9/src/fhir/1_Questionnaire_ECOG12.json#L3)

Reply:
Thanks for catching that - fixed now.